src/utility/api.ts

```diff
@@ -0,0 +1,23 @@
+import axios from "axios";
+import { Task } from "../types/Task.types";
+
+const baseURL: string = process.env.REACT_APP_API_HOST as string;
+
+export const getTasks = async (): Promise<Task[]> => {
+  const response = await axios.get(baseURL);
+  return response.data;
+};
+
+export const addTask = async (task: Task): Promise<Task> => {
+  const response = await axios.post(baseURL, task);
+  return response.data;
+};
+
+export const deleteTask = async (taskId: number): Promise<void> => {
```

Comment:
So right now I've programmed the API to return 204 status code specifically for deletions, since it's considered best practice. What do you think about checking for a success based on the status code? Because if the deletion did not occur as expected the api will return a 400-599 status code. Let's talk error handling tomorrow.

Reply:
Whatever works for you better.
I'm not an API expert, it's up to you what you want to return.
You can also return success = true, whatever you like.

Reply:
Ok, resolving now. I've got an idea :)